Add versioning and reference to criteria mapping sheet to all criteria (#11)
Llannelongue/greenDiSC: Criteria/Bronze/Research Computing Infrastructure - Bronze.md

# Bronze Criteria for Research Computing Infrastructure teams

> You can see an overview of all criteria for Bronze, Silver, and Gold (coming soon!) for Research Groups, Central Teams, and Research Computing Infrastructure Teams [here](https://github.com/Cambridge-Sustainable-Computing-Lab/greenDiSC/blob/main/Criteria/Green%20DiSC%20Criteria%20Overview_03072026.pdf).

Reviewer: ```suggestion
You can see an overview of how the different criteria for Bronze, Silver, and Gold (coming soon!) relate to each other [here](https://github.com/Cambridge-Sustainable-Computing-Lab/greenDiSC/blob/main/Criteria/Green%20DiSC%20Criteria%20Overview_03072026.pdf).
```
